helm/cloud-provider-cloud-director-app/charts/cloud-director-named-disk-csi-driver/values.yaml

```diff
@@ -0,0 +1,43 @@
+basicAuthSecret:
+  create: true # Set to false if you create the secret somewhere else
+  username: ""
+  password: ""
+  refreshToken: ""
+  
+csiConfigmap:
+  site: "" # VCD endpoint with the format https://VCD_HOST. No trailing '/'
+  org: "" # VCD organization name where the cluster should be deployed
+  ovdc: "" # VCD virtual datacenter name where the cluster should be deployed
+  vAppName: "" # "name" field of VCDCluster object in management cluster
+  clusterid: "" # "Infra Id" field of VCDCluster object in management cluster
+  immutable: true
+
+controllerStatefulset:
+  replicas: 1
+  dnsPolicy: Default
+  imagePullPolicy: IfNotPresent
+  nodePluginDaemonset:
+    dnsPolicy: Default
+  csiAttacher:
+    image: k8s.gcr.io/sig-storage/csi-attacher
+    tag: v3.2.1
+  csiProvisioner:
+    image: k8s.gcr.io/sig-storage/csi-provisioner
+    tag: v2.2.2
+  csiPlugin:
+    image: projects.registry.vmware.com/vmware-cloud-director/cloud-director-named-disk-csi-driver

```

Comment:
I would retag the images https://github.com/giantswarm/cloud-provider-cloud-director-app/pull/2/files#r896491893

Reply:
https://github.com/giantswarm/roadmap/issues/1172